lapack/lapack64/lapack64.go

```diff
@@ -19,6 +19,14 @@ func Use(l lapack.Float64) {
 	lapack64 = l
 }
 
+// Tridiagonal represents a tridiagonal matrix using its three diagonals.
+type Tridiagonal struct {
```

Comment:
I added the type here because it exists only in Lapack. An alternative would be in `blas64` where all the other types are.

Reply:
Keeping it here is reasonable.

Reply:
Ok, thanks.